aait/backend/group-2B/Application/Features/Posts/Requests/Queries/GetPostByIdRequest.cs

```diff
@@ -0,0 +1,13 @@
+using MediatR;
+using SocialSync.Application.DTOs.PostDtos;
+
+namespace SocialSync.Application.Features.Posts.Requests.Queries;
+
+public class GetPostByIdRequest : IRequest<GeneralPostDto>
+{
```

Comment:
It would be better if we make Id a property, 
making Id a property aligns with common coding conventions and design patterns. It provides a consistent approach for accessing and manipulating the identifier across different classes and codebases.


Possible change:

`using MediatR;
using SocialSync.Application.DTOs.PostDtos;

namespace SocialSync.Application.Features.Posts.Requests.Queries;

public class GetPostByIdRequest : IRequest<GeneralPostDto>
{
    public int Id { get; set; }
}
`

Reply:
Fixed.